src/Dashboard/CompletedDash.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { createRef } from 'react'
+import { View, ScrollView ,Text, TouchableOpacity,ProgressBarAndroid} from 'react-native'
+import theme, { Card } from '../theme'
+
+
+
+const CompletedDash =()=>{
+  return(
+   
+    <Card
+      
+    variant='elevated'
+    borderRadius='l'
+    marginBottom='l'
+    padding='none'
+    backgroundColor='white'
+    >
+      <View style={{marginBottom:40,}}>
+    <View style={{flexDirection:'row',}}>
+      <Text style={{marginVertical:15,color:theme.colors. darkSilver,fontSize:15}} >Exam Title</Text>

```

Comment:
Try to use Text element from theme, its already created there
Example here : 
import {Text } from '*/theme'
and use
<Text variant='body' marginVertical='s'>some text</Text>